components/ui/code-tab-page/code-tab-page.tsx

```diff
@@ -59,7 +59,8 @@ export function resolveFilePath(
 
   if (fileTree.includes(normalized)) return normalized;
 
-  const requestedExt = path.extname(normalized);
+  const extension = path.extname(normalized);
+  const requestedExt = ['.js', '.jsx', '.ts', '.tsx', '.mjs', '.cjs'].includes(extension) ? extension : '';
```

Comment:
shouldn't it be the opposite?
if the extension will be `.runtime.js` it will be false which I guess it's not what you meant

Reply:
if the extension is `.runtime.js` -> path.extname would be `.js` not `.runtime.js`. It should work as expected?